tests/integration/update_cluster/bastionadditional_user-data/kubernetes.tf

```diff
@@ -942,6 +942,15 @@ resource "aws_security_group" "nodes-bastionuserdata-example-com" {
   vpc_id = aws_vpc.bastionuserdata-example-com.id
 }
 
+resource "aws_security_group_rule" "from-0-0-0-0--0-ingress-icmp-3to4-api-elb-bastionuserdata-example-com" {
```

Comment:
these renames are unfortunate. not only because the apply might cause a temporary loss in network connectivity, but because `terraform` may run into consistency conflicts since we're deleting and creating tf resources that represent the same underlying AWS resource. I've had this happen many times, albeit not with SG rules specifically:

* terraform attempts to delete the rule (old resource) and create the same rule (new resource)
* creation fails because the rule already exists
* deletion succeeds
* terraform doesn't retry the creation because it isnt an error that would normally succeed if retried
* `terraform apply` aborts
* rerun `terraform apply` a second time, the plan reports _only_ creating the new rule. the creation succeeds 

We can always suggest yet another `terraform state mv` (seems to be a regular occurrence with every kOps release these days)